Mathlib/Data/Finset/Card.lean

```diff
@@ -66,16 +66,14 @@ theorem card_le_card : s ⊆ t → s.card ≤ t.card :=
 theorem card_mono : Monotone (@card α) := by apply card_le_card
 #align finset.card_mono Finset.card_mono
 
-@[simp]
-theorem card_eq_zero : s.card = 0 ↔ s = ∅ :=
-  card_eq_zero.trans val_eq_zero
+@[simp] lemma card_eq_zero : s.card = 0 ↔ s = ∅ := card_eq_zero.trans val_eq_zero
```

Comment:
All the changes in this file have already been made in #12526 and should be removed from this PR.

Reply:
(clarified in PM)